hc/accounts/views.py

```diff
@@ -187,6 +185,23 @@ def profile(request):
                                       user=farewell_user).delete()
 
                 messages.info(request, "%s removed from team!" % email)
+        elif "select_allowed_checks" in request.POST:
+            post_data = request.POST   # TODO Clean out
+            post_data = json.dumps(dict(post_data))
+            post_data = json.loads(post_data)
+            check_ids = post_data.get('checks', None)
+            member_id = post_data.get('member_id', None)
+
+            # remove previous entries
+            for allowed_check in MemberAllowedChecks.objects.filter(user_id=member_id[0]):
+                allowed_check.delete()
+
+            # allowed checks selected for user
+            if check_ids:
+                for check_id in check_ids:
+                    check = Check.objects.get(pk=check_id)
```

Comment:
Even more appropriate is MemberAllowedChecks.objects.create(user=member, checks__id=check_id) since we don't require the reference to` allowed_user` for any more reason than to save it